src/index.jsx

```diff
@@ -133,6 +133,7 @@ initialize({
         ENABLE_CHECKLIST_QUALITY: process.env.ENABLE_CHECKLIST_QUALITY || 'true',
         ENABLE_GRADING_METHOD_IN_PROBLEMS: process.env.ENABLE_GRADING_METHOD_IN_PROBLEMS === 'true',
         LIBRARY_MODE: process.env.LIBRARY_MODE || 'v1 only',
+        LIBRARY_SUPPORTED_BLOCKS: (process.env.LIBRARY_SUPPORTED_BLOCKS || 'problem,video,html').split(','),
```

Comment:
@navinkarkera Could you add this in [.env](https://github.com/openedx/frontend-app-authoring/blob/master/.env), [.env.development](https://github.com/openedx/frontend-app-authoring/blob/master/.env.development) and [.env.test](https://github.com/openedx/frontend-app-authoring/blob/master/.env.test)?

Reply:
@ChrisChV Done. https://github.com/openedx/frontend-app-authoring/pull/1378/commits/61e52687cae3b0883766ae1fa20f0b0782d58cf5